core/shared/src/main/scala/zio/NonEmptyChunk.scala

```diff
@@ -262,7 +262,7 @@ object NonEmptyChunk {
    * Constructs a `NonEmptyChunk` from one or more values.
    */
   def apply[A](a: A, as: A*): NonEmptyChunk[A] =
-    nonEmpty(Chunk(a) ++ Chunk.fromIterable(as))
+    nonEmpty(Chunk.single(a) ++ Chunk.fromIterable(as))
```

Comment:
I wonder whether this can be improved a bit more by:
1. If `as.isEmpty` shortcut to `Chunk.single(a)`
2. Otherwise use a ChunkBuilder

Reply:
I optimized it in `NonEmptyChunk.fromIterable`

Reply:
Done